mysql-test/suite/rocksdb/t/i_s_trx_rpl.test

```diff
@@ -0,0 +1,42 @@
+--source include/have_rocksdb.inc
+--source include/master-slave.inc
+
+--disable_warnings
+connection slave;
+--source include/stop_slave.inc
+--enable_warnings
+
+connection master;
+create table t1 (a int) engine=rocksdb;
+--disable_query_log
+--let $aa= 0
+while ($aa < 1000) {
+  eval insert into t1 values ($aa);
+  --inc $aa
+}
+--enable_query_log
+
+connection slave;
+show variables like 'rocksdb_rpl_skip_tx_api';
+--source include/start_slave.inc
+
+--let $it=0
```

Comment:
Replace with wait_for_condition.inc, among other things this loop is spinning without any pause between iterations